test/inductor/test_torchinductor.py

```diff
@@ -13741,6 +13742,50 @@ def forward(
                 )
                 torch._inductor.aot_compile(traced, inputs)
 
+        @skipCUDAIf(not SM90OrLater, "Requires sm90")
+        @requires_gpu()
+        @config.patch(implicit_fallbacks=True)
```

Comment:
I removed the lowering I added in inductor - I realized that implicit_fallbacks actually defaults to `True`, but was set to false in this test suite

Reply:
good point. maybe @ngimel would know? (does grouped_mm only support contiguous inputs?)

Reply:
For grouped_mm we don't other than one of the dimensions should be contiguous, similar to regular mm.

Reply:
@bdhirsh can you add `make_fallback(aten._grouped_mm, require_dense)`

Reply:
@eellison what would be the correct way of handling this for custom ops provided by third-party libraries?

Reply:
For custom ops provided by third-party libraries, we are trying to match the eager strides by default. @zou3519 is going to turn that on (i forget if he did or not yet).

Otherwise we have these tags: https://github.com/pytorch/pytorch/blob/236b08cbf83edcd168aaa384e2603dbf8d2acc2f/aten/src/ATen/native/tags.yaml#L45-L67.

Reply:
Ah interesting, so there's no "require_dense" equivalent?

Reply:
Theres not, we could add one if someone wanted

Reply:
> For custom ops provided by third-party libraries, we are trying to match the eager strides by default. @zou3519 is going to turn that on (i forget if he did or not yet).

This is on by default already